edb/server/compiler/compiler.py

```diff
@@ -1796,6 +1814,11 @@ def _compile_dispatch_ql(
             caps |= enums.Capability.MODIFICATIONS
         return (query, caps)
 
+    elif isinstance(ql, qlast.AnalyzeStmt):
+        query = _compile_ql_analyze(ctx, ql, script_info=script_info)
+        caps = enums.Capability(0)
```

Comment:
We'll probably want some capabilities required here if we ever take this out of dev mode.
Not sure if it matters now, though.